src/tools/interop/idt/capture/ecosystem/play_services/prober.py

```diff
@@ -21,7 +21,9 @@
 
 from . import config
 
-logger = log.get_logger(__file__)
+_LOGGER = log.get_logger(__file__)
+_URLS_FILE = os.path.join(os.path.dirname(__file__), "urls.txt")
+_URLS_FILE_EXISTS = os.path.exists(_URLS_FILE)
```

Comment:
why is this here instead of being inline when we do the check? Is exists too expensive or do we want it run only once? If so this should be `_URLS_FILES_EXISTED_WHEN_MODULE_WAS_LOADED` (and if I name it like that it starts looking odd ... why do I depend on load time? what if it existed then but got removed later?)

Reply:
I guess the idea was that I didn't want variable runtime behavior other than by editing the source (e.g. configs or this file) or through the arparse flags. However, I could see how one might forget this file and want to add it mid capture, so moved to your suggestion; its also safer thanks!